extras/build/material-icons2.js

```diff
@@ -0,0 +1,114 @@
+const packageName = 'material-design-icons'
+const iconSetName = 'Google Material Design Icons'
+
+// ------------
+
+const { copySync } = require('fs-extra')
+const { resolve } = require('path')
+const fetch = require('cross-fetch')
+
+let skipped = {}
+let distFolder = {}
+// const distFolder = resolve(__dirname, `../material-icons`)
+const { extractSvg, writeExports, Queue, sleep, retry } = require('./utils')
+
+const svgFolder = resolve(__dirname, `../node_modules/${packageName}/`)
+const iconNames = {}
+
+const svgExports = {}
+const typeExports = {}
+
+const themeMap = {
+  baseline: '', // filled
+  outlined: '_outlined',
+  round: '_round',
+  // twotone: '_two_tone',
+  sharp: '_sharp',
+}
+
+function downloadIcon(icon) {
+  // console.log(`downloadIcon ${icon.index}: ${icon.name}`)
```

Comment:
Can this be removed or you need it in some way?
If it's needed, can we mark it with a comment explaining why so?

Reply:
Comment removed - thanks